docs/source/nodes_and_pipelines/modular_pipelines.md

```diff
@@ -72,11 +42,71 @@ Running the `kedro pipeline create` command adds boilerplate folders and files f
 
 ```
 
-</details>
+If you want to delete an existing pipeline, you can use `kedro pipeline delete <pipeline_name>` to do so.
+```{note}
+To see the full list of available CLI options, you can run `kedro pipeline create --help` for more information.
+```
+
+## How to structure your pipeline creation
+
+After creating the pipeline with `kedro pipeline create`, you will find template code in `pipeline.py` that you need to fill with your actual pipeline code:
+
+```python
+# src/my_project/pipelines/{{pipeline_name}}/pipeline.py
+from kedro.pipeline import Pipeline, pipeline
+
+def create_pipeline(**kwargs) -> Pipeline:
+    return pipeline([])
+```
+Here, you are creating a function that returns a `Pipeline` class instance with the help of the `pipeline` function. We recommend putting your pipeline creation code into a function because it allows to achieve:
+- Modularity: you can easily reuse this logic across different parts of your project.
```

Comment:
I'd maybe skip this bullet list, or I'd move it to the top of the page, where the concept is explained and justified

Reply:
I changed them with explanation that function should be named `create_pipeline()` for auto-discovery.